### PR TITLE
V1

### PR DESCRIPTION
**Pull request by dharniel45**
## Description

Closes: #XXXX

---

## Reviewers

- @kashishshah 
- @Lagrange-Labs/lsc-core-dev
- @Lagrange-Labs/distributed-system

### CODE REVIEW

File: docs/state-committees/architecture/architecture-overview.mdx [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+Nodes wishing to join the cross-chain State Committees must restake via EigenLayer. A node must provide at least 10 ETH worth of rehypothecated collateral to join the network and must indicate which chain or roll-up they wish to provide attestations for.
+
+Every node in the network must run a containerized validator or watcher of a relevant chain or roll-up. If If an operator manages multiple restaked Ethereum nodes, they can establish a secure RPC connection to a single instance of the chain’s validator, ideally running locally on their system, for all nodes. Once in the network, the attestation node will sign the batch of rollup blocks using their BLS key (BN254 curve).
+
+## Responsibilities of a Cross-Chain State Attester
+
+Each cross-chain state attester is responsible for executing a signature with its BLS (BN254) key on every finalized batch during the attestation periods that it is active for. Each signature is executed on a tuple containing a `batchHeader`, `committeeHeader`. Merkle root of public keys, operators, and aggregated signatures of the `batch`. 
+
+```
```
[the remark this answers is not in the document]
Comment (kashishshah) [reply docs/state-committees/architecture/architecture-overview.mdx:64]: you might also have to add `go` to `additionalLanguages` in `docusaurus.config.ts` file

File: docs/state-committees/architecture/key-components.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+## gRPC Server
+
+The **gRPC Server** is a critical physical component of the architecture, tasked with facilitating communication between attestation nodes. When a batch becomes available, it initiates a round. During this round, the entire batch is proposed to the network, a process referred to as the round state.
+
+This round enables the State Committee to verify the proposed batch through a consensus process. Once 67% of the voting power is amassed for a given round state, this mechanism aggregates all the signatures provided by the operators. These signatures are then attached to the batch, finalizing the verification process.
+
+## Attestation Nodes
+
+**Attestation Nodes**, also referred to as the Client nodes. They are operators in the LSC protocol that participates in aggregate signature generation.
```
Comment (kashishshah) [inline docs/state-committees/architecture/key-components.md:39]:
```suggestion
**Attestation Nodes**, also referred to as the Client nodes or Validator nodes. They are operators in the LSC protocol that participates in aggregate signature generation.
```

File: docs/state-committees/architecture/key-components.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+The State Committee operates a committee of restaked attestation nodes, where each node verifies the accuracy of Layer 1 (L1) data from Ethereum and proposed batch data from EVM Rollup Chains. The protocol utilizes EigenLayer’s Actively Validated Services (AVS), leveraging Ethereum’s shared security. AVS enables LSC Operators to engage in validation tasks, thus enhancing the network’s security and integrity.
+
+Each operator may manage multiple attestation nodes, with the staked amount determining their voting power. The LSC has introduced an Epoch period, currently set at one week, to ensure that an operator’s voting power remains constant during that period. To verify the committed Boneh–Lynn–Shacham (BLS) signature, a committee was established consisting of operators with consistent voting power. A Merkle tree, generated from the attestation node information, embeds its root in the batch structure.
+
+After the verification process, the attestation node uploads the BLS signature of the batch to the server. A key feature of BLS signatures is their ability to be aggregated, allowing for more efficient processing.
+
+The Role of EigenLayer AVS in LSC
```
Comment (kashishshah) [inline docs/state-committees/architecture/key-components.md:49]: remove this line

File: docs/state-committees/quickstart.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
+---
```
Comment (kashishshah) [inline docs/state-committees/quickstart.md:1]: Lets make quickstart the first page. Please remove "Overview" section

File: docs/zk-coprocessor/azuki-lending-quickstart.md [hunk 5]
```diff
@@ -0,0 +1,98 @@
+---
```
Comment (kashishshah) [inline docs/zk-coprocessor/azuki-lending-quickstart.md:1]: I believe we should also have quickstart as the first page for ZK Coprocessor.

File: docs/state-committees/architecture/architecture-overview.mdx [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+- The **Consensus** is responsible for computing aggregate signature and ensuring 67% of total voting power is used to sign the batch in order to consider it as finalized.
+- The **Attestation Nodes** are operators in the LSC protocol that participates in aggregate signature generation. 
+
+<ThemedImage
+  lightSrc="/img/state-committee-light.png"
+  darkSrc="/img/state-committee-dark.png"
+  alt="State Committees Architecture Diagram"
+/>
+
+**State Committee Contracts** - The Lagrange service and Lagrange committee smart contract is deployed on the Ethereum mainnet. The smart contract manages epoch updates, voting power updates and calculates the Merkle tree.  A Merkle tree is computed by hashing individual data blocks to form leaf nodes, then iteratively hashing pairs of nodes to form parent nodes until a single hash, the Merkle root, is obtained. This root summarizes the data, allowing efficient and secure verification of the data’s integrity with minimal information.
```
[the remark this answers is not in the document]
Comment (duncand0nuts) [reply docs/state-committees/architecture/architecture-overview.mdx:48]: I was wondering if a full explanation of merkle tree is necessary here - I think we can link to a description of it elsewhere?

File: docs/state-committees/architecture/architecture-overview.mdx [hunk 7]
```diff
@@ -0,0 +1,105 @@
+---
+id: architecture-overview
+title: "Architecture"
+description: A brief description about State Committees architecture
+author: kashish
```
[the remark this answers is not in the document]
Comment (dharniel45) [reply docs/state-committees/architecture/architecture-overview.mdx:5]: Yes sure

File: docs/state-committees/architecture/key-components.md [hunk 8]
```diff
@@ -0,0 +1,55 @@
+---
+id: key-components
+title: "Key Components"
+description:
```
[the remark this answers is not in the document]
Comment (dharniel45) [reply docs/state-committees/architecture/key-components.md:4]: sorted

File: docs/state-committees/architecture/architecture-overview.mdx (hunk 6 above)
[the remark this answers is not in the document]
Comment (dharniel45) [reply docs/state-committees/architecture/architecture-overview.mdx:48]: @kashishshah asked me to leave some details about how a Merkle tree is computed somewhere in the docs @duncand0nuts

File: docs/state-committees/quickstart.md (hunk 4 above)
Comment (dharniel45) [reply docs/state-committees/quickstart.md:1]: The Overview section sort of provides that introduction to what the protocol is all about. It is very good for media outlet looking for quick accurate info on the LSC @kashishshah

Comment (dharniel45) [reply docs/state-committees/quickstart.md:1]: I was thinking we retain the overview section particularly for media outlet or new potential users looking for a quick overview of what the product in the protocol is all about.

Comment (kashishshah) [reply docs/state-committees/quickstart.md:1]: agreed

**Review by kashishshah — APPROVED** [review]

**Pull request merged**